units/systemd-pcrextend.socket

```diff
@@ -8,7 +8,7 @@
 #  (at your option) any later version.
 
 [Unit]
-Description=TPM2 PCR Extension (Varlink)
+Description=TPM PCR Measurements
```

Comment:
hmm so far i relatively carefully made sure to always say TPM2, and never just TPM. Here you are reversing this. 

Which I am not totally against, but if you do this, please do it comprehensively, i.e. in doc/ and man/ as well.

Reply:
We don't support TPM1, so it seems better to be more precise? We did have people open issue because they were trying this stuff with a TPM 1.2 in the past

Reply:
I'm pretty sure we should lose the distinction between TPM and TPM2 in titles and names. OTOH, we should be very clear in the descriptions that TPM1.x is not supported.

Generally,  exposing versions of things too much becomes a problem fairly quickly. If we get TPM 2.1 or 3.0 or whatever, we certainly wouldn't want to retitle all the targets. I'll submit a follow-up PR to change the name in more places.

Reply:
Or to look at this from another side: users know that they have "a tpm" in their machine. Windows and most user-facing documentation calls this "tpm". The fact that it's a specific version of the standard is really a low-level detail.